detect_secrets/plugins/keyword.py

```diff
@@ -45,6 +45,7 @@
     'private_key',
     'secret',
     'secrete',
+    'token',
```

Comment:
I am kind of ambivalent about this, when we [first wrote the keyword detector, `token` was a keyword](https://github.com/Yelp/detect-secrets/commit/d912ced2e7a7607f06794e3d916b10e52536bd5c#diff-1b8ec3b1cef86dbae9081fdea381e7fbL40), however the signal-to-noise ratio wasn't as good as the other keywords, it was too noisy.

Inflexibility is an anti-pattern though, which is one of the reasons we added [a keyword exclude regex](https://github.com/Yelp/detect-secrets/pull/132), so the static `FALSE_POSITIVES` dictionary could be added to and  customized for things specific to each users' codebase.

We should probably do the same thing for keyword detector tokens, though it is tricky work, because we will have to write and read it from the baseline file. #146 and #151 are examples of issues stemming from work like this.

This would also not require other keyword additions like #148 to be blocked on us doing a ton of internal testing.

Reply:
I'm wondering what kind of internal testing do you guys carry to accept / reject a new keyword? We haven't built up a large test code base yet, which we probably should. I'm curious how big is your test code base, and what's process for you to evaluate false positive ratio?

Reply:
At least for the keyword detector, we made the regexes super loose so it would be noisy, found all the true positives stemming from it, and trimmed the regexes down best we could to catch as little false-positives as possible. A decent amount of manual work was involved with this.

It's still a little loud now, I plan on working on it a little more, but have been doing some other projects recently.

For testing, we ran it on all of our largest codebases, I'm not sure of the exact LoC, but probably a few hundred thousand or more.

p.s. [Bumped ✊ the version today, btw 👍](https://github.com/Yelp/detect-secrets/commit/4e9555a60e9444d5153b90fe9d0d6b688e14e414?short_path=4ac32a7#diff-4ac32a78649ca5bdd8e0ba38b7006a1e)